rfcs/core_user_interface_improvement/20230314-project_proposal_core_ui_improvement.md

```diff
@@ -0,0 +1,41 @@
+# Project proposal
+
+## Reviewers
+
+- [ ] @dhruvkb
+- [ ] @zackkrida
+
+## Project summary
+
+Update the visual style of VButton component in both Design Library and
+Storybook components to set a consistent baseline for all UI components.
+
+## Goals
+
+Search Experience
+
+## Requirements
+
+<!-- Detailed descriptions of the features required for the project. Include user stories if you feel they'd be helpful, but focus on describing a specification for how the feature would work with an eye towards edge cases. -->
+
+## Success
+
+- VButton code component follows the design documentation specs.
+- Design Library documents the VButton-based components following Storybook
+  documentation.
+
+## Participants and stakeholders
+
+<!-- Who is working on the project and who are the external stakeholders, if any? -->
+
+- Design: @panchovm
+
+## Infrastructure
+
+<!-- What infrastructural considerations need to be made for this project? If there are none, say so explicitly rather than deleting the section. -->
+
+## Marketing
+
+There is no need for marketing or any public communication.
+
+## Required implementation plans
```

Comment:
Let's delete this heading if no plans are required.

Reply:
So there is no need to assess and put down the technical impact of this project? I don't know who will be the dev participants working on this to ping them and ensure they had sufficient time to reflect on this.

Reply:
I was asking you to remove the section if you thought the project didn't need a technical plan. I wasn't suggesting that it didn't need planning. 

However, as @obulat shared elsewhere, this will probably be a single pull request so it probably does not need planning.

I'll keep my original suggestion that we remove this heading.

Reply:
> I was asking you to remove the section _if you thought the project didn't need a technical plan_.

I don't have the dev expertise to respond to this.

Reply:
👍 In the future we can add something to the proposals like the below:

```md
## Required implementation plans

I have a question for reviewers and the members of @WordPress/openverse-frontend. Does this work need technical planning before it can begin?
```

I was just confused by the empty heading.